src/app/pages/SearchSitter/SearchSitter.module.css

```
@@ -0,0 +1,26 @@
+.container {
+  height: 100vh;
+  display: grid;
+  grid-template-rows: auto 1fr;
+  gap: 1em;
+}
+
+.header {
+  margin-left: 1em;
+}
+
+.main {
+  display: grid;
```

```suggestion
  display: flex;
  flex-direction: column;
```
Should solve the problem with the image